### ISSUE TITLE
ghp-import: python: bad interpreter: No such file or directory

### ISSUE THREAD

**Issue by Azd325** [davisp/ghp-import#49]
I tried pelican and used the command `make github` on python 3.5 and i got this error.
On python2.7 is everything fine.

``` bash
~/p/t/timkl.xyz (content) $ make github
pelican /home/tim/projects/timkl.xyz/timkl.xyz/content -o /home/tim/projects/timkl.xyz/timkl.xyz/output -s /home/tim/projects/timkl.xyz/timkl.xyz/publishconf.py 
Done: Processed 11 articles, 0 drafts, 0 pages and 0 hidden pages in 0.31 seconds.
ghp-import -m "Generate Pelican site" -b master /home/tim/projects/timkl.xyz/timkl.xyz/output
/bin/sh: /home/tim/.virtualenvs/pelican/bin/ghp-import: python: bad interpreter: No such file or directory
```

I found this related to my problem but it didn't help me.
https://github.com/getpelican/pelican/issues/1274

**Comment by standage** [comment]
I had the same exact problem when installing `ghp-import` with pip: it worked in Python 2.x, but Python 3.x gave me a bad interpreter message. After a bit of tinkering, I got it to work by replacing `#!python` with `#!/usr/bin/env python` on the first line of the file.

Since it's such an easy fix, I was about to submit a pull request. But it looks like the code in this Github repository has already been fixed. So the problem seems to be that the Python package index does not have the latest version of `ghp-import`. Until the developers issue another release, it looks like you'll need to install from Github rather than `pip`.
